pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -172,26 +221,51 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 	}
 	// all secret events before this signal must be processed before we're marked "ready"
 	c.queue.Add(initialSyncSignal)
-
+	time.AfterFunc(features.RemoteClusterTimeout, func() {
+		c.remoteSyncTimeout.Store(true)
```

Comment:
I think we need to sound some alarms when this happens - logs + metrics

Reply:
added